test/suite/esy.test.js

```diff
@@ -7,15 +7,14 @@ const fs = require("fs-extra");
 const { Uri } = vscode;
 
 let root = path.dirname(path.dirname(__dirname));
-let fixtureSrcDir = path.join(root, "fixtures");
+let fixtureDir = path.join(root, "fixtures");
```

Comment:
Previously, these fixtures would be copied over to TMPDIR. But now that e2e need settings.json with the path of the esy toolchain specified in it relative to the project itself, e2e tests will setup the project in the work tree and run tests there.